hack/test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# $1 - Specifies distribution - RHEL7/CentOS7
+# $2 - Specifies Ruby version - 2.0
+
+# Array of all versions of Ruby
+declare -a VERSIONS=(2.0)
+
+OS=$1
+VERSION=$2
+
+if [ -z ${VERSION} ]; then
+  # Build all versions
+  dirs=${VERSIONS}
+else
+  # Build only specified version on Ruby
+  dirs=${VERSION}
+fi
+
+for dir in ${dirs}; do
+  IMAGE_NAME=openshift/ruby-${dir//./}-${OS}
+  echo ">>>> Testing ${IMAGE_NAME}"
+
+  pushd ${dir} > /dev/null
+
+  if [ "$OS" == "rhel7" ]; then
```

Comment:
Specifically for this block I'm fine putting it into hack/common.sh and just include that file to avoid duplications. @jhadvig my concerns were more related about having one big 'triggers.sh' with all code ;-) Making this more re-usable as @bparees mentioned is good thing.

But you don't have to do that right now ;-)